homeassistant/components/fyta/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "fyta",
+  "name": "FYTA",
+  "codeowners": ["@dontinelli"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/fyta",
+  "integration_type": "hub",
+  "iot_class": "cloud_polling",
+  "requirements": ["fyta_cli==0.2.3"]
```

Comment:
Sanity check on `0.2.3`:
- https://github.com/dontinelli/fyta_cli/blob/v0.2.3/src/fyta_cli/fyta_client.py#L9 requests are not used and can, therefore, be removed
- https://github.com/dontinelli/fyta_cli/blob/v0.2.3/pyproject.toml#L12-L14 are builtin directly in python and don't need to be specified. Or do you really want to use the ones published on pypi? `asyncio` has not updated since 2015
- You use `pytz` to specify the  timezone and default to UTC, which is already builtin directly in python. See https://docs.python.org/3/library/datetime.html#datetime.UTC

Reply:
Please keep this conversation open and the PR in draft until `fyta_cli` is bumped in the manifest.json to a version that fixes the above findings.
Even `v0.3.2` is already released it is not used by this integration yet.